### PR TITLE
[MRG] Modified examples for better readability

### PR DESCRIPTION
**Pull request by pavanramkumar**
The rationale for removing `poisson` and `softplus` examples is that the cheatsheet clearly explains the difference.

### CODE REVIEW

File: pyglmnet/utils.py [hunk 1]
```diff
@@ -72,7 +73,20 @@ def log_likelihood(y, yhat, distr):
 
 def tikhonov_from_prior(PriorCov, n_samples):
     """Given a prior covariance matrix, returns a Tikhonov matrix"""
+    """Given a prior covariance matrix, returns a Tikhonov matrix"""
     [U, S, V] = np.linalg.svd(PriorCov, full_matrices=False)
-    Tau = np.dot(np.diag(1. / np.sqrt(S)), U)
-    Tau = 1. / np.sqrt(np.float(n_samples)) * Tau / Tau.max()
+
+    S_ratio = S / S.max()
+    threshold = 0.0001
```
Comment (jasmainak) [inline pyglmnet/utils.py]: dumb question, why do we have a `threshold` here and why is it hard coded? can you point me to the equations somewhere?

**Comment by jasmainak** [comment]
@pavanramkumar merge if you're happy!

File: pyglmnet/utils.py (hunk 1 above)
Comment (pavanramkumar) [inline pyglmnet/utils.py]: thanks! good eye! have made it a parameter now and defined it within the function. it specifies a cutoff of where to truncate singular values based on the ratio of largest to smallest singular value. 

same thing is done in MNE :)

Comment (jasmainak) [inline pyglmnet/utils.py]: But you're doing it to solve a system of equations? I think `scipy.solve` does it better ... but never mind, I realized that `scipy` is not a dependency ...

**Pull request merged**